README.md

```diff
@@ -27,7 +27,11 @@ The format for companies is the following:
 Short description.
 ```
 
-### Agriculture & Food
+### Vegetation, Agriculture & Food
```

Comment:
Can you change the title and link on the table of contents as well? It should be:

```
[Vegetation, Agriculture & Food](#vegetation-agriculture--food)
```